solana/solana-ibc/programs/solana-ibc/src/tests.rs

```diff
@@ -64,7 +66,7 @@ macro_rules! make_message {
 #[test]
 #[ignore = "Requires local validator to run"]
 fn anchor_test_deliver() -> Result<()> {
-    let authority = Rc::new(Keypair::new());
+    let authority = Rc::new(read_keypair_file("../../keypair.json").unwrap());
```

Comment:
why not generating a new keypair?

Reply:
I am reading the keypair because i wanted the same keypair in tests and validator executable.